dart/biomechanics/DynamicsFitter.cpp

```diff
@@ -8750,7 +8770,9 @@ DynamicsFitter::DynamicsFitter(
     mSilenceOutput(false),
     mDisableLinesearch(false)
 {
+  std::cout << "Initial mass: " << mSkeleton->getMass() << std::endl;
   mSkeleton->setGroupMasses(mSkeleton->getGroupMasses());
+  std::cout << "Reset mass: " << mSkeleton->getMass() << std::endl;
```

Comment:
Did you mean to keep this cout statements?

Reply:
I think we can ditch them.